app/notifications/process_notifications.py

```diff
@@ -41,7 +41,7 @@ def create_content_for_notification(template, personalisation):
 
 def check_placeholders(template_object):
     if template_object.missing_data:
-        message = "Missing personalisation: {}".format(", ".join(template_object.missing_data))
+        message = "Missing personalisation for template ID {}: {}".format(template_object.id, ", ".join(template_object.missing_data))
```

Comment:
I also added this extra information when I tried to debug a validation issue and didn't have the required template ID on missing personalisation. I had to infer from previous logging statements.